client/src/components/Modals/ClaimPitch/index.tsx

```diff
@@ -121,6 +121,16 @@ const ClaimPitchModal: FC<ClaimPitchProps> = ({
     if (!isValidForm()) {
       return;
     }
+    const teams: string[] = []
+    checkboxes.forEach(function (selected, team) {
+      if (selected) {
+        teams.push(team);
+      }
+    })
+    const userInfo = {
```

Comment:
Please update line 140 when calling submitPitchClaim with the new teams, I think the current code does not pass anything in. Possibly consider changing the parameter type to something that would mimic the structure of userInfo?